[debug] [2018-10-09T15:03:45.334Z] ----------------------------------------------------------------------
[debug] [2018-10-09T15:03:45.336Z] Command:       /usr/local/bin/node /usr/local/bin/firebase serve
[debug] [2018-10-09T15:03:45.336Z] CLI Version:   5.0.1
[debug] [2018-10-09T15:03:45.336Z] Platform:      darwin
[debug] [2018-10-09T15:03:45.336Z] Node Version:  v8.9.0
[debug] [2018-10-09T15:03:45.336Z] Time:          Tue Oct 09 2018 18:03:45 GMT+0300 (EEST)
[debug] [2018-10-09T15:03:45.336Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-10-09T15:03:45.347Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-10-09T15:03:45.347Z] > authorizing via signed-in user
[debug] [2018-10-09T15:03:45.347Z] [iam] checking project adelin-ade0022 for permissions ["firebase.projects.get"]
[debug] [2018-10-09T15:03:45.349Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/adelin-ade0022:testIamPermissions  

[debug] [2018-10-09T15:03:47.123Z] <<< HTTP RESPONSE 200
[debug] [2018-10-09T15:03:47.124Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/adelin-ade0022  
 
[debug] [2018-10-09T15:03:48.439Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/Users/adelinsofrag/Desktop/Git Projects/FirebaseTesting/project22'...
[info] 
[debug] [2018-10-09T15:03:49.764Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:113:21)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:9:29)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:17:21
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:7)
    at Command._action (/usr/local/lib/node_modules/firebase-tools/commands/serve.js:47:12)
[error] 
[error] Error: An unexpected error has occurred.
